src/macaron/slsa_analyzer/checks/detect_malicious_metadata_check.py

```diff
@@ -42,6 +42,8 @@
 class MaliciousMetadataFacts(CheckFacts):
     """The ORM mapping for justifications in pypi heuristic check."""
 
+    # pylint: disable=unsubscriptable-object
```

Comment:
Could you also include the comment about temporarily silencing pylint here?

Reply:
Oh, I was supposed to remove that one.